### DISCUSSION TITLE
灌水，我没有梯子，为什么只有晚上可以上，白天上不了？

### DISCUSSION THREAD

**Discussion by 1451374815** [share121/inter-knot#956] [其他]
为什么只有晚上可以上，白天上不了？

**Comment by Mayf1ower** [comment]
这感觉挺随缘的，不挂梯子确实有时候也能上Github，可能是墙对访问GitHub这个站点管的不严吧

**Reply by 1451374815** [reply]
有没有免费的电脑端梯子啊，

**Comment by capacitor1** [comment]
墙了，但没有完全墙。

**Reply by 1451374815** [reply]
有没有免费的电脑端梯子啊，

**Reply by ARequest** [reply]
watt toolkit咯 微软商店里有好像

**Comment by Qiling-XwX** [comment]
进来了，但是没有完全进来

**Comment by qwq1949** [comment]
可以用这个加速器→ [网站是免费的](https://steampp.net/)
好像github没被墙 是限带宽了 白天可能人多？吧

**Comment by apiaoapiao** [comment]
半夜两点，我现在能直接连
